packages/Card/src/components/Separator/Separator.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import * as sc from "./Separator.styles";
+
+const propTypes = {
+  /** Size of the separator (min-height). */
+  size: PropTypes.oneOf(ShirtSizes.DEFAULT),
+};
+
+const defaultProps = {
+  size: ShirtSizes.SMALL,
+};
+
+const Separator = props => {
+  return (
+    <sc.separatorStyles data-pka-anchor="card.separator" {...props}>
+      &nbsp;
+    </sc.separatorStyles>
+  );
+};
+
+Separator.displayName = "Separator";
+Separator.propTypes = propTypes;
+Separator.defaultProps = defaultProps;
+
+export default Separator;
```

Comment:
So this has 3 sizes or whatever.  Let's say small is 10px, medium is 20px, large is 30px.
What happens when a designer inevitably wants 24px of separation?

And are we supposed to use it like `<br />`s?  That doesn't feel right:

    ...content...
    <Separator />
    <Separator />
    <Separator />
    ...more content...

Reply:
And what about when a card has more content than the one beside it? i dont think this would be possible with so much control over the content:

![Screen Shot 2020-05-25 at 3 26 45 PM](https://user-images.githubusercontent.com/23224777/82846369-5e74fa00-9e9d-11ea-9dca-7d3214920ae7.png)

The first card would be cut off the “Last updated by” line.
You support flexibility in the header and footer, but not the body.  In my opinion the Card's API should be simpler, not micro-managing as much:

    <Card>
      <Header />
      <Body>
        <Title />
        ...any nodes that i want...
      </Body>
      <Footer />
    </Card>

I bring this up for discussion with @acl-services/ux-devs, not requesting immediate changes.

Reply:
I agree.  I don't think a `<Separator>` component is the right approach. We can add `margin-bottom` via the parent (`<Content>`) component, or the `<Content>` (or `<Body>` in Jamie's example) could contain another container that the consumer supplies and would have min-height (to match heights) or margin or whatever it wants.

Reply:
Agreed, separator not the right approach.

Reply:
I actually think the separator it's the correct approach. But with the ability to add custom height. My proposal would be something like:

`<Separator space={Separator.small} />` you only need one separator.

or 

`<Separator space={45} />`

But definitely like oscar said once, won't going to die in this hill. 

So please either let Karen and Kaan if we should remove this component or add my modification.

But lets take a decision so Jamie can start using this component :)

Reply:
If we only need to use 1 Separator, then i'm okay for now. Can we add that prop in instead of removing it completely. Should be quick to fix?

Reply:
Wont a single separator still have problem with spacing between two sections of text?  In English the two items will fit correctly, but in German they wont (since the text could be a line longer, the space would need to be smaller).

Reply:
ohh I understand now what do you mean Jamie, make sense to remove the Separator for now and let the consumer deal with this depending on their needs. make sense?

Reply:
It makes sense to me